lib/network/modules/components/nodes/util/NodeBase.js

```diff
@@ -43,6 +55,13 @@ class NodeBase {
     labelModule.adjustSizes(this.margin)
   }
 
+  /**
+   *
+   * @param {CanvasRenderingContext2D} ctx
+   * @param {Number} angle
+   * @returns {Number}
+   * @private
```

Comment:
OK tough call. This is used virtually and gets overridden in child classes. It doesn't feel like private to me.